tlf/id.go

```diff
@@ -257,3 +257,45 @@ func MakeRandomID(t Type) (ID, error) {
 	}
 	return id, nil
 }
+
+// MakeIDFromTeam makes a deterministic TLF ID from a team ID and an epoch
+// representing how many times a new TLF has been needed for this
+// team.  Returns NullID on failure.
```

Comment:
What's the scenario where tracking this is necessary?

Reply:
Tracking what, the epoch?  It's for if we ever have to manually reset an implicit team TLF (due to a KBFS bug, say), without wanting to reset the entire implicit team.

Reply:
Makes sense, thanks.